telegram-bot/src/main/kotlin/com/ajaxproject/telegrambot/service/CurrencyService.kt

```diff
@@ -12,30 +12,26 @@ import reactor.core.publisher.Mono
 
 @Component
 class CurrencyService(
-    private val currencyExchangeCacheableRepository: CurrencyRepository,
+    private val currencyRedisRepository: CurrencyRepository,
```

Comment:
```suggestion
    private val currencyRepository: CurrencyRepository,
```

It's some repository, we don't know which one, actually, we don't care which one